__tests__/app.test.js

```diff
@@ -200,28 +206,62 @@ describe("POST /api/articles/:article_id/comments", () => {
       .post("/api/articles/3/comments")
       .send({})
       .expect(400)
-      .then(({ body }) => {
-        expect(body.message).toBe("missing fields username and content");
+      .then(({ body: { message } }) => {
+        expect(message).toBe("missing fields username and content");
       });
   });
-  test("404: Responds with an error message when given a non-existent id for the article", () => {
+  test("404: Responds with an error message when the user given does not exist", () => {
     return request(app)
-      .get("/api/articles/9000/comments")
+      .post("/api/articles/1/comments")
+      .send({
+        username: "notUser",
+        body: "Test comment",
+      })
       .expect(404)
-      .then(({ body }) => {
-        expect(body.message).toBe("article does not exist");
+      .then(({ body: { message } }) => {
+        expect(message).toBe("User does not exist");
       });
   });
-  test("404: Responds with an error message when the user given does not exist", () => {
+  test("400: Responds with an error message when given an invalid id", () => {
     return request(app)
-      .post("/api/articles/1/comments")
+      .post("/api/articles/newspaper-article/comments")
+      .expect(400)
       .send({
-        username: "snake_survived",
-        body: "Kept you waiting, huh?",
+        username: "butter_bridge",
+        body: "I am a test comment",
       })
-      .expect(404)
-      .then(({ body }) => {
-        expect(body.message).toBe("User does not exist");
+      .then(({ body: { message } }) => {
+        expect(message).toBe("Bad request - article Id can only be a number");
+      });
+  });
+});
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("200: Responds with an increased number of votes from the object received", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: 10 })
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article.votes).toBe(110);
+      });
+  });
+  test("200: Responds with a decreased number of votes from the object received", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: -10 })
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article.votes).toBe(90);
+      });
+  });
+  test("400: Responds with an error message when given an invalid id", () => {
+    return request(app)
+      .patch("/api/articles/ramen-article")
+      .send({ inc_votes: 5 })
+      .expect(400)
+      .then(({ body: { message } }) => {
+        expect(message).toBe("Bad request - article Id can only be a number");
```

Comment:
These are all great tests Javier and I can see you've added a 404 now on the following PR 👍 
There is just one more test you can do though, what happens if we sent across an incorrect body (e.g. if inc_votes is not a number) ?